helm-charts/strimzi-kafka-operator/README.md

```diff
@@ -6,9 +6,9 @@ See our [website](https://github.com/strimzi/strimzi-kafka-operator) for more de
 
 ## Introduction
 
-This chart bootstraps the Strimzi Cluster Operator, Roles, Service Accounts, and Strimzi Custom Resource Definitions 
-for running [Apache Kafka](https://kafka.apache.org/) on [Kubernetes](http://kubernetes.io) cluster using the 
-[Helm](https://helm.sh) package manager.
+This chart bootstraps the Strimzi Cluster Operator Deployment, Roles, Role Bindings, Service Accounts, and 
```

Comment:
Technically its Cluster Roles and Cluster Role Bindings. I'm not sure whether its so important to change it, but it might tell the users what to expect in terms of rights required to deploy it.

Reply:
Cool.  Updated :+1: